ads/pubexchange.md

```diff
@@ -0,0 +1,44 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# PubExchange
+
+## Example
+
+### Basic
+
+```html
+<amp-embed width="640" height="320"
+    heights="(max-width:480px) 400%, (max-width:650px) 100%, 75%"
+    layout="responsive"
+    type="pubexchange"
+    data-publication="test_publication"
+    data-moduleId="below_content"
+    data-moduleNum="2626"
+    data-test="true">
+</amp-embed>
+```
+
+## Configuration
+
+For semantics of configuration, please see ad network documentation.
+
+Supported parameters:
+
+- data-publication *(**mandatory**)* - Shortcode identifying publication provided by PubExchange account manager
+- data-moduleId *(**mandatory**)* - Shortcode identifying module provided by PubExchange account manager
```

Comment:
FYI: `data-moduleId` will be converted to `data-moduleid`. The usual approach is request `data-module-id` and it will be converted to `data-moduleId`. 
I would suggest the other way, but up to your decision.